src/compiler/checker.ts

```diff
@@ -4617,12 +4617,10 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
             ? location
             : (isModuleDeclaration(location) ? location : location.parent && isModuleDeclaration(location.parent) && location.parent.name === location ? location.parent : undefined)?.name ||
                 (isLiteralImportTypeNode(location) ? location : undefined)?.argument.literal ||
-                (isInJSFile(location) && isJSDocImportTag(location) ? location.moduleSpecifier : undefined) ||

```

Comment:
Does it matter that we aren't checking `isInJsFile` any more? Would we resolve an `@import` inside a ts file now?

Reply:
Confirmed by changing the test to .ts instead. I'll add that as importTag22.ts and merge the PR.